corpus/templates/athenaeum/home.html

```diff
@@ -0,0 +1,417 @@
+                <p>
+                The Athenaeum talks are conducted every year as part of the IEEE week from 1st October to 6th October. 
+                Speakers from various fields talk about interesting topics, emerging technologies and amazing research.
+                Each society invites speakers of more or less their own domain
+                </p>
+            </section>
+            <section id="workshops" class="py-5 px-5">
+                <h2>Events</h2>
+                <div class="flex flex-col md:flex-row justify-between content-center gap-4">
+                    <div class="card bg-base-100 shadow-xl basis-1/2 border">
+                        <div class="card-body">
+                            <h2 class="card-title">Tech it Easy Quiz!</h2>
+                            <strong>4th October, 2024</strong>
+                            <div class="card-actions">
+                                <a href="https://rb.gy/vjfnmb" class="btn btn-primary no-underline">
+                                    <strong>Quiz Link</strong>
+                                </a>
+                                <a href="https://drive.google.com/drive/folders/1k3VXRj_B8IvBBtiQgOSYMKrCXEvupe9P"
+                                   class="btn btn-secondary no-underline">
+                                    <strong>Quiz Resources</strong>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <section id="speakers" class="py-5 px-5">
+                <h2>Speakers</h2>
+                <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+                    <div class="card bg-base-100 shadow-xl basis-1/2 border">
+                        <figure>
+                            <img src="{% static 'img/athenaeum/Kshitij_Anand.png' %}" alt="Kshitij Anand"
+                                 class="h-96 rounded-md"/>
+                        </figure>
+                        <div class="card-body">
+                            <h2 class="card-title">Signals, Information Theory and Quantitative Finance</h2>
+                            <span>By Kshitij Anand</span>
+                            <span>IIT Bhubaneswar</span>
+                            <strong>5th October, 2024 at 6:00PM</strong>
+                            <div class="card-actions">
+                                <a href="https://rb.gy/ciaa6q" class="btn btn-primary no-underline">
+                                    <strong>Session Link</strong>
+                                </a>
+                                <a href="https://drive.google.com/drive/folders/1Nnq8WbO17oH7kHY13Vj4XgYYCUXuWVhY?usp=drive_link"
+                                   class="btn btn-secondary no-underline">
+                                    <strong>Session Resources</strong>
+                                </a>
```

Comment:
since session hasn't happened yet, better to not display this button right?

Reply:
Right. Should I just remove all links? We can't keep updating it all the time and most of the sessions are done anyways

Reply:
I can add resources link later

Reply:
better imo to add later, since its more of an informative page rn, or else for events until yest u can add